apps/cms/config/sync/user.role.publisher.yml

```diff
@@ -0,0 +1,9 @@
+uuid: a6375354-2d5f-4632-b53e-619040dec631
+langcode: en
+status: true
+dependencies: {  }
+id: publisher
+label: Publisher
+weight: -5
+is_admin: null
+permissions: {  }
```

Comment:
What is this role for? It has no permissions, so looks suspicious.

Reply:
It is only used for the Consumer (content entity) scope.
As we are just granting access to Publisher currently, it has no permissions.

It could be used as a placeholder role, for what the Publisher UI could do with Drupal without being current user specific. I cannot see a use case for now so we might just remove it.

Reply:
And also adjust the Consumer creation in the deploy hook.

Reply:
The role can confuse someone... like me 😅 

As an admin, I'd like to know what is this role for and when to assign it to users. Could you answer this?

Reply:
Sorry for the late reply.
The original reasoning behind is similar to the `Gatsby Build` role with the `Gatsby` user. It's a way to limit how Publisher can interact with Drupal with this Consumer.

But as 
- we don't have any `Publisher` user currently
- we have another authentication system for Gatsby to source from Drupal
- there is nothing specific that `Publisher` can do as we are using the current user permissions check - specifically the `access publisher` permission, e.g. for the `administrator` role here https://github.com/AmazeeLabs/silverback-template/pull/86/files/f3fecfcbe3eb2e600b744bb4785377c0209b3842#diff-dda0826d23d77d3c618b19281ec0a74bb7d14e8f13e6e85d021bb0f77c1b1a2eR32

let's just remove it as it can also be confusing / we might be tempted to assign this role to users instead of the permission.

Reply:
Removed here
- https://github.com/AmazeeLabs/silverback-template/pull/86/commits/96fa474761657c49cb408c4888e6dc4ac1208541
- and not using it as part of the Consumer setup https://github.com/AmazeeLabs/silverback-template/pull/86/commits/6e1fc325392c86cf78886a8e5ef6903f4269e6f5#diff-36927e5378571a60a719b3904a4d26d00bcc8137c4d512d2cd4e9f01dcfbc686R47-R53 - as opposed to https://github.com/AmazeeLabs/silverback-mono/blob/development/packages/composer/amazeelabs/silverback_gatsby/modules/silverback_gatsby_oauth/silverback_gatsby_oauth.deploy.php#L20-L23 and https://github.com/AmazeeLabs/silverback-mono/blob/development/packages/composer/amazeelabs/silverback_gatsby/modules/silverback_gatsby_oauth/silverback_gatsby_oauth.deploy.php#L59-L61

Also moving Consumers setup from `silverback-mono` https://github.com/AmazeeLabs/silverback-mono/pull/1448 to the `custom` module as it's template specific.